Sources/Conformance/main.swift

```diff
@@ -60,32 +60,39 @@ func buildResponse(serializedData: Data) -> Conformance_ConformanceResponse {
         return response
     }
 
-    let parsed: ProtobufTestMessages_Proto3_TestAllTypes?
+    var msgType: SwiftProtobuf.Message.Type
```

Comment:
I think you can make this a `let` since the compiler can determine that it was definitely assigned before first use.